### PR TITLE
Feature: Make main menu configurable

### PR DESCRIPTION
**Pull request by wutschel**
## Description

Closes https://github.com/xbmc/Official-Kodi-Remote-iOS/issues/306.

Details:
- Each main menu entry can be switched on/off in Kodi App settings (default is ON for each)
- Added localized strings for English and German
- Minor refactoring when building the menu (iPad per default now also has NowPlaying)

## Summary for release notes

Feature: Make main menu configurable

### CODE REVIEW

**Comment by wutschel** [comment]
I am adding this PR as work-in-progress to allow testing. It should be added only after the bigger code style and AppDelegate rework PRs to avoid problems rebasing those.

**Comment by wutschel** [comment]
Rebased to master.

**Comment by wutschel** [comment]
Rebased to master.

File: XBMC Remote/AppDelegate.m [hunk 1]
```diff
@@ -4579,38 +4579,51 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
...
-        [mainMenuItems addObject:menu_LiveTV];
-        [mainMenuItems addObject:menu_Remote];
         self.windowController = [[ViewControllerIPad alloc] initWithNibName:@"ViewControllerIPad" bundle:nil];
         self.windowController.mainMenu = mainMenuItems;
         self.window.rootViewController = self.windowController;
     }
     return YES;
 }
 
+-(BOOL)isMenuEntryEnabled:(NSUserDefaults*)userDefault menu:(NSString*)menuItem {
```
Comment (kambala-decapitator) [inline XBMC Remote/AppDelegate.m:4623]: doesn't make much sense to pass `NSUserDefaults` singleton as parameter

**Review by kambala-decapitator — CHANGES_REQUESTED** [review]
looks good, just one change to make

File: XBMC Remote/AppDelegate.m (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/AppDelegate.m:4623]: Like this:

```
-(BOOL)isMenuEntryEnabled:(NSString*)menuItem {
    id menuEnabled = [[NSUserDefaults standardUserDefaults] objectForKey:menuItem];
    return (menuEnabled == nil || [menuEnabled boolValue]);
}
```

Comment (wutschel) [reply XBMC Remote/AppDelegate.m]: Decided to go with first option.

**Comment by wutschel** [comment]
Rebased to master and updated following the review comment.

**Pull request merged**